src/se/key.rs

```diff
@@ -1,4 +1,4 @@
-use crate::errors::serialize::DeError;
+use crate::errors::serialize::SeError;
```

Comment:
In all other places, you used this path to error. Let's here too, for consistency:
```suggestion
use crate::se::SeError;
```